lib/resources/Invoices.js

```diff
@@ -14,6 +13,17 @@ module.exports = StripeResource.extend({
     urlParams: ['id'],
   }),
 
+  listLines: stripeMethod({
+    method: 'GET',
+    path: '{id}/lines',
+    urlParams: ['id'],
+  }),
+
+  listLinesUpcoming: stripeMethod({
```

Comment:
In other libraries we do this by passing `upcoming` as the id.

Reply:
I would actually think this'd be `listUpcomingLines`...

Reply:
I'm also uncomfortable-but-peaceful with an `'upcoming'` id if that's what we do in other libs (and if it has not caused user confusion/sadness)

Reply:
I renamed it `listUpcomingLineItems`, which is consistent with what we do in .NET: https://stripe.com/docs/api/invoices/upcoming_invoice_lines?lang=dotnet.

We should also add this to other languages because having users pass a magic ID as a string is terrible.